lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -254,62 +285,62 @@
                 weight: 5,
             }
         id:
-            description: the record id
+            description: The record ID.
             returned: success
             type: str
             sample: f9efb0549e96abcb750de63b38c9576e
         locked:
-            description: No documentation available
+            description: No documentation available.
             returned: success
             type: bool
             sample: False
         meta:
-            description: No documentation available
+            description: No documentation available.
```

Comment:
Same question here - does this mean Cloudflare doesn't document this field? Or we don't (on purpose?)???

Reply:
You should ask the maintainer. If you're really up to it, open an issue :-)